package.json

```diff
@@ -44,28 +44,28 @@
     "@storybook/preview-api": "^8.6.12",
     "@storybook/react": "^8.6.12",
     "@storybook/react-vite": "^8.6.12",
-    "@storybook/test-runner": "^0.22.0",
+    "@storybook/test-runner": "^0.22.1",
     "@testing-library/jest-dom": "^6.6.3",
     "@testing-library/react": "^16.3.0",
     "@testing-library/user-event": "^14.6.1",
     "@types/history": "^4.7.11",
     "@types/jest": "^29.5.14",
-    "@types/node": "^22.14.0",
-    "@types/react": "^19.1.0",
-    "@types/react-dom": "^19.1.0",
+    "@types/node": "^22.15.34",
+    "@types/react": "^19.1.8",
+    "@types/react-dom": "^19.1.6",
     "@types/react-highlight": "^0.12.8",
     "@types/react-router": "^5.1.20",
     "@types/react-router-dom": "^5.1.1",
     "@types/react-transition-group": "^4.4.12",
     "@types/wicg-file-system-access": "^2023.10.6",
-    "cspell": "^9.0.1",
+    "cspell": "^9.1.2",
     "jest": "^29.7.0",
     "jsdom-testing-mocks": "^1.13.1",
-    "msw": "^2.7.6",
-    "msw-storybook-addon": "^2.0.4",
+    "msw": "^2.10.2",
```

Comment:
Some types are broken after this update and I'm yet to fix them, feel free to pick this up.